database/git_index.ksy

```diff
@@ -0,0 +1,150 @@
+meta:
+  id: git_index
+  title: Git index
+  license: CC0-1.0
```

Comment:
Git is GPLed, so are its docs. Git maintainer is pro-GPL, since he has called applying the techniques used to reverse engineer proprietary software in the most licensially-clean way to git as "necessary effort" and has refused to initiate relicensing of the docs into more permissive license.

Reply:
No, this is not how it works. I read the e-mail thread and you misunderstood the developers: https://www.spinics.net/lists/git/msg371734.html

```
GPL copyright protects the expression of the document, but the
copyright protects only the expression, and does not protect the
underlying format itself and the idea behind it.  So I do not see a
need to relicense the documentation text at all.
```
So no, I am not going to relicense this file either.